src/pages/DataTable/DataTable.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import './DataTable.css';
+import { Link } from '@reach/router';
+import Names from '../../names.json';
+import firebase from 'firebase';
+
+class TableRowItem extends Component {
+    render() { 
+        return (
+            <tr>
+                <td>{this.props.name}</td>
+                <td>{this.props.jacobpick ? "True" : ""}</td>
+                <td>{this.props.partner != "" ? this.props.partner : this.props.pickedby}</td>
+            </tr>
+    );}
+}
+
+class DataTable extends Component {
+
+    state = {
+        students: [],
+    }
+
+    componentDidMount() {
+        let ref = firebase.database().ref();
+        ref.on('value', snapshot => {
+            this.setState({
+                students: snapshot.val()
+            });
+        });
+    }
+
+    render() {
+        return (
+            <table>
+                <tbody>
+                    <tr>
+                        <th>Name</th>
+                        <th>Jacob Pick?</th>
+                        <th>Partner</th>
+                    </tr>
+                    {this.state.students.map((student) => {
+                            console.log(student);
```

Comment:
Might not need the `console.log` statement anymore ;P

Reply:
Ha, true!